CPO-13 Dynamic configuration (#336)
hazelcast/hz-docs: docs/modules/configuration/pages/configuring-programmatically.adoc

= Configuration Object

> :description: You can use a configuration object (config object) to configure members and clients with any Hazelcast client or the Java member API. To create new instances of members or clients, it's enough to pass the config object to those instances.

Author: ```suggestion
:description: You can use a configuration object (config object) to configure members and clients with any Hazelcast client or the Java member API. To create new instances of members or clients, pass the config object to those instances.
```

Config config = Config.loadFromString(yaml, properties);
HazelcastInstance hz = Hazelcast.newHazelcastInstance(config);
----

== Dynamic Configuration in Java

> If you use Java, you can add configuration for some data structures at runtime, using the `add*Config()` methods of supported features.

Reviewer: Resolved.

Config config = Config.loadFromString(yaml, properties);
HazelcastInstance hz = Hazelcast.newHazelcastInstance(config);
----

> == Updating Member Configuration Files at Runtime

Author: ```suggestion
== Updating Member Configuration at Runtime
```

> You can add configuration for some data structures at runtime. See xref:dynamic-data-structure-configuration.adoc[].

Author: ```suggestion
You can add configuration for some supported features at runtime. See xref:dynamic-data-structure-configuration.adoc[].
```
